SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -52,6 +52,388 @@ void CDriver::PreprocessPythonInterface(CConfig** config, CGeometry**** geometry
   }
 }
 
+/////////////////////////////////////////////////////////////////////////////
+/* Functions related to farfield flow variables.                           */
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
Can you change these to the more common `/*--- comment ---*/`

Reply:
You mean just something like this instead? Removing all the `/////` from top and bottom as well?

`/*--- Functions related to farfield flow variables. ---*/`